src/config-hardware/printers.md

```diff
@@ -0,0 +1,124 @@
+# Setting up Printers
+
+Most printers are plug-and-play in Pop!\_OS. Add a network or USB connected printers in `Settings` ➞ `Printers`. Clicking `Add a Printer` will automatically search for any connected printers. Printers can also be added manually by clicking `Additional Printer Settings`.
+
+![Printer Settings Screen](/images/printers/printer-settings.png)
+
+---
+
+## CUPS and Printer Compatibility
+
+Pop!\_OS uses the Common Unix Printing System (CUPS) print server to manage print jobs, print queues, and recognizes connected printers. While CUPS recognizes a wide range of printer models, printer functionality may vary. Check if your printer is compatible using [OpenPrinter.com](https://www.openprinting.org/printers).
+
+## Checking Printer Status in CUPS
+
+Open this web page to look at the CUPS (Common Unix Printing System) configuration and status window:
+
+[localhost:631](http://localhost:631)
```

Comment:
Not sure this needs to be on its own line.

It might be worth using HTML instead of Markdown in order to force this to open in a new tab (with `target="_blank"`).